### DISCUSSION TITLE
Putting .sol file in Contracts folder does not work

### DISCUSSION THREAD

**Discussion by AliSeraji** [Cyfrin/foundry-full-course-cu#2658] [Q&A]
Hi everyone, 

I want to ask how to solve the problem with the Zksync plugin, because even after putting my contract file in the `contracts` folder, and compiling, I still get the same problem that says `No contracts ready for deployment yet, compile a solidity contract.`
There is also no error in the console. It is just shown in the Deploy section.
 
 I want to know how you are dealing with the problem. 
 
Additionally, I am using the Remix app on Windows 11 and I am learning through the course in Cyfrin .
 
<img width="210" alt="image" src="https://github.com/user-attachments/assets/f2263fbc-c611-47d3-9c7f-5c1ca5894dca">

**Comment by EngrPips** [comment]
Hello @AliSeraji, You should not bother yourself too much with the `zksync` part of the course if things don't work appropriately for you. Working with `zksync` isn't very smooth at the moment, and that is why Patrick keeps repeating that `zksync` part of the course is optional.

**Reply by AliSeraji** [reply]
Thanks a lot for this advice. 
And sorry for replying late.

**Reply by EngrPips** [reply]
No worries at all.

**Comment by chainflash** [comment]
Write the contracts folder in lowercase letters and make sure you wrote it correctly
